petpal/accounts/management/commands/set_reverse_relation.py

```diff
@@ -0,0 +1,14 @@
+from django.core.management.base import BaseCommand
+from comments.models import Comment
+
+class Command(BaseCommand):
+    help = 'Set reverse relation for comments'
+
+    def handle(self, *args, **options):
+        comments = Comment.objects.all()
+
+        for comment in comments:
+            comment.notification.comment_set.add(comment)
+            comment.save()
+
+        self.stdout.write(self.style.SUCCESS('Reverse relation set successfully'))
```

Comment:
I think this file would better exist in the comments app since it is a command directly related to comments.
`accounts/management/commands/` -> `comments/management/commands`